### PR TITLE
get rid of all baserom calls in graphics.s

### PR DESCRIPTION
**Pull request by ProjectRevoTPP**

### CODE REVIEW

File: data/pokenav.s [hunk 1]
```diff
@@ -1511,10 +1511,10 @@ gUnknown_086230E4:: @ 86230E4
 	.4byte sub_81CC95C
 
 gUnknown_086230F8:: @ 86230F8
-	obj_tiles gUnknown_08DC9208, 0x800, 6
+	obj_tiles gHoennMapZoomIcons_Gfx, 0x800, 6
 
 gUnknown_08623100:: @ 8623100
-	obj_pal gUnknown_08DC91E8, 11
+	obj_pal gHoennMapZoomIcons_Pal, 11
 	obj_pal NULL, 0
```
Comment (Diegoisawesome) [inline data/pokenav.s]: null_obj_pal

File: data/slot_machine.s [hunk 2]
```diff
@@ -277,8 +332,17 @@ gUnknown_085A8520:: @ 85A8520
...
-	.incbin "baserom.gba", 0x5a8544, 0x48
+	.4byte gUnknown_08DCF170, 0
+	.4byte gUnknown_08DCF190, 1
+	.4byte gUnknown_08DCF1B0, 2
+	.4byte gSlotMachineReelTime_Pal, 3
+	.4byte gUnknown_08DCF1F0, 4
+	.4byte gUnknown_08DCF210, 5
+	.4byte gUnknown_08DCF230, 6
+	.4byte gUnknown_08DCF1F0, 7
+	.4byte 0, 0
```
Comment (Diegoisawesome) [inline data/slot_machine.s]: null_obj_pal

File: data/trainer_card.s [hunk 3]
```diff
@@ -210,19 +214,19 @@ gUnknown_08571454:: @ 8571454
...
-	obj_tiles gUnknown_08570BA8, 0x100, 0
-	obj_tiles gUnknown_08570D00, 0x400, 1
-	obj_tiles gUnknown_08DC1394, 0x380, 2
+	obj_tiles gFrontierPassCursor_Gfx, 0x100, 0
+	obj_tiles gFrontierPassMapCursor_Gfx, 0x400, 1
+	obj_tiles gFrontierPassMedals_Gfx, 0x380, 2
 
 gUnknown_085714D4:: @ 85714D4
 	obj_tiles gUnknown_08570C44, 0x100, 4
 	obj_tiles NULL, 0, 0
```
Comment (Diegoisawesome) [inline data/trainer_card.s]: null_obj_tiles

File: data/trainer_card.s [hunk 4]
```diff
@@ -210,19 +214,19 @@ gUnknown_08571454:: @ 8571454
...
-	 obj_pal gUnknown_08DC1694, 1
-	 obj_pal gUnknown_08DC16B4, 2
-	 obj_pal gUnknown_08DC16D4, 3
+	 obj_pal gFrontierPassCursor_Pal, 0
+	 obj_pal gFrontierPassMapCursor_Pal, 1
+	 obj_pal gFrontierPassMedalsSilver_Pal, 2
+	 obj_pal gFrontierPassMedalsGold_Pal, 3
 	 obj_pal gUnknown_0856FB7C, 4
 	 obj_pal gUnknown_0856FB9C, 5
 	 obj_pal NULL, 0
```
Comment (Diegoisawesome) [inline data/trainer_card.s]: null_obj_pal

File: data/slot_machine.s [hunk 5]
```diff
@@ -241,8 +241,63 @@ gUnknown_085A8264:: @ 85A8264
 gUnknown_085A82CC:: @ 85A82CC
 	.incbin "baserom.gba", 0x5a82cc, 0x68
 
+	@ 50 F2 DC 08 00 02 00 00
```
[the remark this answers is not in the document]
Comment (ProjectRevoTPP) [reply data/slot_machine.s]: those comments are for the below incbin ive dumped. oops

**Pull request merged**